README.md

```diff
@@ -749,9 +749,9 @@ Examples:
 
 Source Location | example expression  | notes
 ---|:---|:---|
-Request parameter      | `$request.id`        | Request parameters MUST be declared in the `parameters` section of the parent operation or they cannot be evaluated.
-Deep Request parameter | `$request.user.uuid`   | In methods which accept nested object payloads, `.` may be used to denote traversal of an object.
-Response value         | `$response.uuid`       |  In methods which return payloads, references may be made to portions of the response body or the entire body.
+Request parameter      | `$params.id`        | Request parameters MUST be declared in the `params` section of the parent operation or they cannot be evaluated.
+Deep Request parameter | `$params.user.uuid`   | In methods which accept nested object payloads, `.` may be used to denote traversal of an object.
+Response value         | `$results.uuid`       |  In methods which return payloads, references may be made to portions of the response body or the entire body.
```

Comment:
need to fix up this "Response value" text about `response body` and `entire body`. any suggestions?

Reply:
gah, this whole section needs help lol

Reply:
I wouldn't be opposed to removing links, and PR'ing it back in having rethought how it works within the context of open rpc. ATM its straight copy of open api, and there are probably some improvements to be made in there.